src/assets/translations/knb.json

```diff
@@ -2107,7 +2107,7 @@
                   "verse": "6"
                 },
                 {
-                  "text": "Jézus ezt mondta neki: »Az is írva van: ‘Ne kísértsd az Urat, Istenedet‘«.",
+                  "text": "Jézus ezt mondta neki: »Az is írva van: ‘Ne kísértsd az Urat, Istenedet’«.",
```

Comment:
Fixing the response of szentiras.hu API:
https://szentiras.hu/api/idezet/Mt4,7/KNB

I am not sure what is in the text Bible.